pkgs/tools/virtualization/google-guest-agent/default.nix

```diff
@@ -0,0 +1,40 @@
+{ buildGoModule, fetchFromGitHub, lib, coreutils }:
+
+buildGoModule rec {
+  pname = "guest-agent";
+  version = "20211214.00";
+  owner = "GoogleCloudPlatform";
+  rev = version;
```

Comment:
why are these defined here? they are uniquely used by `fetchFromGithub`

Reply:
Good catch, guess I just copied that from somewhere. Fixed.